update/.ansible-lint

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
can we somehow use it in CI to lint ansible?

Reply:
I'm afraid it'll take long to ship this PR. Maybe as a separate task?

Reply:
I believe this step runs the linter. It's part of CI as well.

https://github.com/percona/pmm/blob/fd3be9302d5aef9ddf815e92bad107468f3cd111/update/Makefile#L76

Reply:
Interesting... I don't see where we trigger it, do you have a reference? I searched for `make check` throughout the pipelines.

Reply:
Or is it part of an action?